### DISCUSSION TITLE
Add notes to Mark as Pass.rst

### DISCUSSION THREAD

**Discussion by steierj** [steierj/ReadtheDocsFork#1] [Ideas]
* add a note to Mark as Pass SOP that assemblies should be re-annotated with LIMS/FIMS just as a precaution before marking as Pass in LIMS. 
* Also add note on Mark as Submitted to this page. - newly adopted step!
* Also add note on Mark as Tentative to this page - also new

**Comment by redmondn** [comment]
I think this should also be covered when we meet Chris/John - I want to establish exactly when both of these options should/shouldn't be used!

**Reply by steierj** [reply]
"Mark as passed/submitted **date**" data field addition to LIMS?
